extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js

```diff
@@ -99,7 +99,11 @@ RdfPrefixesManager.prototype._hasPrefix = function(name){
 * Some utility functions
 */
 RdfPrefixesManager.isPrefixedQname = function(qname){
-	return qname.match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
+        if ((qname.split(':').length -1) < 3){
```

Comment:
So what happens if a URL contains more than 1 colon in the path?

Reply:
Then it splits the string by colon and then checks with the regex but only against the two first values in the first list joined by a colon. The regex is checking for prefixes i dont think they need the full string